evaluate/train_image.py

```diff
@@ -1,115 +1,115 @@
+    model = L.Classifier(predictor)
     in_size = 224
 else:
     raise ValueError('Invalid architector:{}'.format(args.predictor))
-model = L.Classifier(predictor)
+
 
 if args.gpu >= 0:
     cuda.get_device(args.gpu).use()
     model.to_gpu()
 optimizer = O.SGD()
+optimizer.use_cleargrads()
 optimizer.setup(model)
 
 xp = cuda.cupy if args.gpu >= 0 else numpy
 
-start_iteration = 0 if args.cache_level is None else -1
-
-forward_time = 0.0
-backward_time = 0.0
-update_time = 0.0
-print('iteration\tforward\tbackward\tupdate (in seconds)')
-for iteration in six.moves.range(start_iteration, args.iteration):
-    if args.gpu >= 0:
-        cache.clear_cache(args.cache_level)
-
-    # data generation
-    data = numpy.random.uniform(-1, 1,
-                                (args.batchsize, in_channels, in_size, in_size)).astype(numpy.float32)
-    data = chainer.Variable(xp.asarray(data))
-    label = numpy.zeros((args.batchsize,), dtype=numpy.int32)
-    label = chainer.Variable(xp.asarray(label))
-
-    # forward
-    with timer.get_timer(xp) as t:
-        loss = model(data, label)
-    forward_time_one = t.total_time()
-
-    # backward
-    with timer.get_timer(xp) as t:
-        loss.backward()
-    backward_time_one = t.total_time()
 
-    # parameter update
-    with timer.get_timer(xp) as t:
+# data generation
+total_iteration = args.dry_run + args.iteration
+shape = (total_iteration, args.batchsize, in_channels, in_size, in_size)
+data = xp.random.uniform(-1, 1, shape)
+data = data.astype(numpy.float32, copy=False)
+label = xp.random.randint(0, label_num, (total_iteration, args.batchsize))
+label = label.astype(numpy.int32, copy=False)
+
+# dry run
+for iteration in six.moves.range(args.dry_run):
+    print('Dry Run\t{}'.format(iteration))
+
+    loss = model(data[iteration], label[iteration])
+    model.cleargrads()
+    loss.backward()
+    optimizer.update()
+
+# evaluation
+with timer.get_timer(xp) as t:
+    for iteration in six.moves.range(args.iteration):
+        print('Iteration\t{}'.format(iteration))
+        if args.gpu >= 0:
+            cache.clear_cache(args.cache_level)
+
+        idx = iteration + args.dry_run
+        loss = model(data[idx], label[idx])
+        model.cleargrads()
+        loss.backward()
         optimizer.update()
-    update_time_one = t.total_time()
 
-    if iteration < 0:
-        print('Burn-in\t{}\t{}\t{}'.format(forward_time_one, backward_time_one, update_time_one))
-    else:
-        print('{}\t{}\t{}\t{}'.format(iteration, forward_time_one, backward_time_one, update_time_one))
-        forward_time += forward_time_one
-        backward_time += backward_time_one
-        update_time += update_time_one
 
-forward_time /= args.iteration
-backward_time /= args.iteration
-update_time /= args.iteration
+total_time = t.total_time()
+averate_gime = total_time / args.iteration
 
-print('Mean\t{}\t{}\t{}'.format(forward_time, backward_time, update_time))
+print('Total Time\t{}'.format(total_time))
```

Comment:
Consider printing the output in a similar format as the corresponding deepmark benchmark. That is, print the average time per iteration and also print the number of examples/second.